src/main/resources/db/migration/common/V1_352__not_null_constraints_notifications.sql

```diff
@@ -0,0 +1,11 @@
+update notification set created_ts = current_timestamp where created_ts is null;
+alter table notification alter column created_ts set not null;
+
+update notification set created_by = 0 where created_by is null;
```

Comment:
may need to add a clause in case the user account doesnlt exist any more:-

e.g.
 or not exists (select 1 from user_account ua where ua.usr_id = theTable.created_by);

Reply:
done